topics/opengl/index.md

```diff
@@ -0,0 +1,12 @@
+---
+created_by: Silicon Graphics
+display_name: OpenGL
+logo: opengl.png
+released: January 1992
+short_description: OpenGL is a software interface that allows a programmer to communicate with graphics hardware.
+topic: opengl
+related: api, c, simulation
+url: https://www.opengl.org/
+wikipedia_url: https://en.wikipedia.org/wiki/OpenGL
+---
+OpenGL (Open Graphics Library) is the computer industry's standard API for defining 2-D and 3-D graphic images. OpenGL allows new hardware innovations to be accessible through the API via the OpenGL extension mechanism. It is used in CAD, virtual reality, video games and flight simulation.
```

Comment:
You could add `(Computer-aided design)` after "CAD".

Can you add a comma before `and flight simulation`? See https://github.com/github/explore/blob/master/docs/styleguide.md#commas.